packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -150,6 +150,16 @@ export interface ISemanticColors {
    */
   variantBorder: string;
 
+  /**
+   * Hover color of border that provides contrast between an element, such as a card, and an emphasized background.
+   */
+  variantBorderHovered: string;
+
+  /**
+   * Background color for default/empty state graphical elements, eg icons, placeholder graphics, etc.
```

Comment:
hey actually that's a good idea, "defaultStateBackground" might be a better name

Reply:
I would add some specific examples, like "empty seats" or "placeholder background when image hasn't loaded yet" or "empty section that needs user to fill in content"